includes/googleMap.php

```diff
@@ -0,0 +1,10 @@
+<!-- Importing google api --> 
+<script 
+   src="//maps.googleapis.com/maps/api/js?v=3.exp&libraries=drawing<?php echo (isset($GOOGLE_MAP_KEY) && $GOOGLE_MAP_KEY?'&key='.$GOOGLE_MAP_KEY:''); ?>">
```

Comment:
Would improve readability to add spacing in the logic for the ternary 
operator here.
Also, per coding standards doc:
> Recommended: add spaces before and after periods when concatenating in php

Reply:
Thanks for the reminding didn't catch that in the coding standards